flow/workflows/qrep_flow.go

```diff
@@ -473,6 +480,9 @@ func QRepWaitForNewRowsWorkflow(ctx workflow.Context, config *protos.QRepConfig,
 		}
 
 		logger.Info("QRepWaitForNewRowsWorkflow: continuing the loop")
```

Comment:
this log should be moved below if-then-return

seems a little odd, why do we not wait for new rows when doing full refreshes?

Reply:
There are always going to be new rows since it's full refresh so this is just to force it to wait for the requested wait time (`WaitBetweenBatchesSeconds`)